### ISSUE TITLE
What should our tagline be?

### ISSUE THREAD

**Issue by hdrdavies** [dwyl/dwyl-site#112]
At current it is 'dwyl is a community of people using technology to solve real problems'

Personally I feel this is a bit too wordy. My suggestion, to capture the same sentiment would be: 'dwyl uses tech to solve real problems'... but unfortunately this doesn't capture the fact that we're awesome people! :smile:

**Comment by iteles** [comment]
@mantagen You had some ideas on this?

**Comment by mantagen** [comment]
My thoughts are that the tagline wants to send this message:
"We are the sort of coders who you want to employ"

**Comment by mantagen** [comment]
@iteles I feel like this would benefit from being opened up to this wider community. Gitter?

**Comment by hdrdavies** [comment]
"dwyl solves your real-life problems"... "dwyl solves your problems"

**Comment by iteles** [comment]
@hdrdavies Still no mention of tech in there - we could be a bunch of psychics with crystal balls... :wink:

**Comment by hdrdavies** [comment]
I thought we were...? :crystal_ball:

**Comment by iteles** [comment]
I love that there's an emoji for that.

**Comment by tunnckoCore** [comment]
> Still no mention of tech in there - we could be a bunch of psychics with crystal balls...

ahahahaha :+1: made my evening, lol

**Comment by tunnckoCore** [comment]
What about _"dwyl - awesome community, solved problems"_ i'm not sure for the dash and da colon. Also not sure for "real" word between solved and problems. 

Actually words "problems" and "real" are not okey with me, i don't know why exactly :D

**Comment by markwilliamfirth** [comment]
Closing due to redesign
